src/usr/local/www/diag_packet_capture.php

```diff
@@ -512,7 +514,17 @@ function hostport_array_fixer($items,$type) {
 	}
 
 	if (in_array($proto, $protos)) {
-		$matches[] = fixup_not(str_replace('carp', 'proto 112', $proto));
+		if ($proto == 'carp') {
+			$matches[] = fixup_not(str_replace('carp', 'proto 112', $proto));
+		} else {
+			if ($fam == 'ip') {
+				$matches[] = fixup_not(str_replace('ospf', 'ip[9] == 89', $proto));
```

Comment:
Why is this necessary? Running with `proto ospf` catches OSPF and OSPFv3 (output shortened):
```
: tcpdump -i vmx0 ip6 and proto ospf
08:28:09.061963 IP6 fe80::20d:b9ff:fe33:f71 > ff02::5: OSPFv3, Hello, length 52
08:28:09.062241 IP6 fe80::20c:29ff:fef8:3bdc > ff02::5: OSPFv3, Hello, length 52
08:28:09.062920 IP6 fe80::20c:29ff:fe45:256 > ff02::5: OSPFv3, Hello, length 52

: tcpdump -i ovpns4 ip and proto ospf
08:28:17.229238 IP 10.4.103.2 > ospf-all.mcast.net: OSPFv2, Hello, length 48
```

Reply:
right, it's unnecessary
fixed